packages/webchat/src/store/index.ts

```diff
@@ -252,16 +258,14 @@ class RootStore {
   /** Fetches the list of conversation, and update the corresponding config values */
   @action.bound
   async fetchConversations(): Promise<void> {
-    const { conversations, recentConversationLifetime, startNewConvoOnTimeout } = await this.api.fetchConversations()
+    const conversations = await this.api.fetchConversations()
 
     runInAction('-> setConversations', () => {
-      if (!conversations.length) {
+      if (!conversations?.length) {
         this.view.showBotInfo()
       }
 
-      this.config.recentConversationLifetime = recentConversationLifetime
-      this.config.startNewConvoOnTimeout = startNewConvoOnTimeout
```

Comment:
These two configurations are actually used often. There could be a single endpoint on messaging's side where you can get the bot's configuration, webchat config and these two configs